tests/data-builders-kotlin/src/main/graphql/schema.graphqls

```diff
@@ -18,6 +18,17 @@ type Query {
   listOfListOfAnimal: [[Animal]!]
   unusedType: UnusedType
   node: Node
+  product: Product
+  promo: Promo
+}
+
+type Promo {
+  product: Product!
+}
+
+type Product {
+  name: String!
+  price: Float!Ò
```

Comment:
Is `Ò` a typo?

Reply:
This is 100% a typo. Fix is there: https://github.com/apollographql/apollo-kotlin/pull/4522. Thanks for the heads up!